src/locales/cy/translation.json

```diff
@@ -368,50 +368,50 @@
+          "yes": "Parhau i ddileu fy manylion",
+          "no": "Mynd yn ôl i'm manylion presennol"
         }
       }
     },
     "pyiConfirmDeleteDetails": {
-      "title": "TEMP Are you sure you want to delete your details and prove your identity again?",
-      "header": "TEMP Are you sure you want to delete your details and prove your identity again?",
+      "title": "Ydych chi'n siwr eich bod am ddileu eich manylion a phrofi eich hunaniaeth eto?",
+      "header": "Ydych chi'n siwr eich bod am ddileu eich manylion a phrofi eich hunaniaeth eto?",
       "content": {
-        "paragraph1": "TEMP Deleting the details saved in your GOV.UK One Login will not affect any other government account you may have such as Government Gateway or Universal Credit.",
+        "paragraph1": "Ni fydd dileu eich GOV.UK One Login yn effeithio ar unrhyw gyfrifon eraill y llywodraeth sydd gennych, fel Porth y Llywodraeth neu Gredyd Cynhwysol.",
         "formRadioButtons": {
-          "yes": "TEMP Yes, delete my details",
-          "no": "TEMP No, cancel and go back to my current details"
+          "yes": "Ydw, dileu fy manylion",
+          "no": "Na, canslo a mynd yn ôl at fy manylion cyfredol "
         }
       }
     },
     "pyiDetailsDeleted": {
-      "title": "TEMP You have deleted the details in your GOV.UK One Login",
-      "header": "TEMP You have deleted the details in your GOV.UK One Login",
-      "content": {
-        "paragraph1": "TEMP You've been sent a confirmation email.",
-        "paragraph2": "TEMP You can now prove your identity again using your new details.",
-        "paragraph3": "TEMP You'll need:",
-        "list1": "TEMP photo ID with the name you want GOV.UK One Login to use",
-        "list2": "TEMP to tell us where you've lived for the last 3 months",
-        "paragraph4": "TEMP Find out <a class=\"govuk-link\" target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://www.gov.uk/using-your-gov-uk-one-login/proving-your-identity\">what photo ID you can use to prove your identity (opens in new tab)</a>.",
-        "submitButtonText": "TEMP Continue to prove your identity again"
+      "title": "Rydych chi wedi dileu'r manylion yn eich GOV.UK One Login",
+      "header": "Rydych chi wedi dileu'r manylion yn eich GOV.UK One Login",
+      "content": {
+        "paragraph1": "Rydych wedi derbyn e-bost cadarnhau.",
+        "paragraph2": "Nawr gallwch brofi eich hunaniaeth eto gan ddefnyddio'ch manylion newydd.",
+        "paragraph3": "Byddwch angen:",
+        "list1": "ID gyda llun gyda'r enw rydych chi ei eisiau GOV.UK One Login ei ddefnyddio ",
+        "list2": "dweud wrthym ble rydych chi wedi byw am y 3 mis diwetha",
+        "paragraph4": "Darganfyddwch <a class=\"govuk-link\" target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://www.gov.uk/using-your-gov-uk-one-login/proving-your-identity\">pa ID gyda llun y gallwch ei ddefnyddio i brofi eich hunaniaeth (agor mewn tab newydd)</a>.",
```

Comment:
Should all the text here be inside the \<a> tag?

Reply:
Again, not according to the designs